src/lib/components/index.js

```diff
@@ -0,0 +1 @@
+export * from "./jsonEditorComponents";
```

Comment:
there is no need for this, put all the components in the current components directory. There is no need to introduce another folder

Reply:
Resolved by using direct imports